.mailmap

```diff
@@ -419,6 +419,7 @@ Peter Marshall <p.s.marshall0@gmail.com> <petermarshall@chromium.org>
 Peter Marton <email@martonpeter.com> <peter@risingstack.com>
 Peter Paugh <ppaugh@chariotsolutions.com>
 Phillip Johnsen <johphi@gmail.com> <phillip.johnsen@finn.no>
+Pietro Marchini <pietro.marchini94@gmail.com>
```

Comment:
If the email you use to commit is the same email that listed on the README then you don't need to add an entry in this file 😄

Reply:
puuurfect, gonna remove it 😁

Reply:
done